### PR TITLE
[MIRROR-FIX] "Nerfs" Hyperzine (Discussion Needed)

### PR DESCRIPTION
**Pull request by ItsSelis**
This actually buffs Hyperzine (increasing it from 1 to 1.2).

Additional Comment from original PR:
> Also makes psilocybin actually hallucinogenic and removes a very annoying hallucination interaction with airlocks.

This PR should close #12972

### CODE REVIEW

**Comment by Heroman3003** [comment]
do we even want it changed either way?

**Comment by ItsSelis** [comment]
> do we even want it changed either way?
> 
> 
> 
> Also this PR seems to include a bunch of new random drugs because why?

Im not sure, I mean, the original PR was still open unlike many of the other ones so I figured it might be a wanted change.

The random drugs were also included in the PR but are most likely due to them being added in the past. I put them into a separate commit to easily remove them again if they are not wanted either.

**Comment by Heroman3003** [comment]
> > do we even want it changed either way?
> > Also this PR seems to include a bunch of new random drugs because why?
> 
> Im not sure, I mean, the original PR was still open unlike many of the other ones so I figured it might be a wanted change.
> 
> The random drugs were also included in the PR but are most likely due to them being added in the past. I put them into a separate commit to easily remove them again if they are not wanted either.

The ones that were kept open have had updates to them within last month or were created in that period as new ruling is to close everything older than a month.

**Comment by ItsSelis** [comment]
> > > do we even want it changed either way?
> > > Also this PR seems to include a bunch of new random drugs because why?
> > 
> > 
> > Im not sure, I mean, the original PR was still open unlike many of the other ones so I figured it might be a wanted change.
> > The random drugs were also included in the PR but are most likely due to them being added in the past. I put them into a separate commit to easily remove them again if they are not wanted either.
> 
> The ones that were kept open have had updates to them within last month or were created in that period as new ruling is to close everything older than a month.

Oh, my bad. Yeah I can remove the random added drugs. Then it can be decided to either merge or close this, depending if it is wanted or not :v

**Comment by Cameron653** [comment]
This is able to be merged so a decision should be made instead of having it sit.

**Comment by TankTheBirb** [comment]
Hyperzine buff is pretty trivial and won't really affect anything. Having the hallucinogen cause hallucinations might be a funny tool here and there. I'd say add it.

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -646,11 +646,6 @@ About the new airlock wires panel:
 					return
 			else /*if(src.justzap)*/
 				return
-		else if(user.hallucination > 50 && prob(10) && src.operating == 0)
```
Comment (Cameron653) [inline code/game/machinery/doors/airlock.dm:649]: From what I've obtained, this should be kept.

And I kind of agree. It's funny.

File: code/modules/reagents/reagents/drugs.dm [hunk 2]
```diff
@@ -143,6 +143,7 @@
 
 	M.druggy = max(M.druggy, 30)
 
+	var/drug_strength = 20
```
Comment (Cameron653) [inline code/modules/reagents/reagents/drugs.dm:146]: This is all good.

File: code/modules/reagents/reagents/medicine.dm [hunk 3]
```diff
@@ -609,7 +609,7 @@
 	..()
 	if(prob(5))
 		M.emote(pick("twitch", "blink_r", "shiver"))
-	M.add_chemical_effect(CE_SPEEDBOOST, 1)
+	M.add_chemical_effect(CE_SPEEDBOOST, 1.2)
```
Comment (Cameron653) [inline code/modules/reagents/reagents/medicine.dm:612]: I personally feel like this is fine but others might disagree. Also hyperzine only really works in increments of 0.5 from what I've been told. I'd say to just bump it back down to only 1.0

**Review by Cameron653 — CHANGES_REQUESTED** [review]
I'd say to re-add the airlock stun and hit the hyperzine back down to 1.0. Possibly open another PR and see if the hyperzine change is wanted still.

**Comment by Cameron653** [comment]
Closing due to the above PR being merged.

**Pull request closed without merging**